src/pages/es/core-concepts/sharing-state.md

```diff
@@ -0,0 +1,697 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Compartir Estado
+i18nReady: true
+setup: |
+  import Tabs from '../../../components/tabs/Tabs'
+  import UIFrameworkTabs from '~/components/tabs/UIFrameworkTabs.astro'
+  import LoopingVideo from '~/components/LoopingVideo.astro'
+---
+
+Al construir tu proyecto con la [islands architecture / hidratación parcial](/es/concepts/islands/), puede que te hayas topado con este problema: **Quiero compartir estado entre mis componentes.**
```

Comment:
What do you think about replace "islands architecture" with "arquitectura de islas"?

Reply:
he visto que hemos traducido esto como: arquitectura de la isla, arquitectura de islas, y algunos lo hemos dejando en ingles. Seria bueno consolidar en una opcion.

Reply:
Oh yeah, I thought we were using the term without being translated but we aren't 😓 I'll change it!

Reply:
Arquitectura de islas como propone @diegognt me parece bien, qué les parece a los demás?